components/sheet/ImportCSVSheet.tsx

```
@@ -35,29 +43,151 @@ type ImportCSVSheetProps = {
+      errors.push(
+        `Row ${index + 1}: Missing required fields (merchant, amount, date, or currency)`
+      );
+    }
+
+    if (amount && isNaN(Number(amount))) {
+      errors.push(`Row ${index + 1}: Amount must be a valid number`);
+    }
+
+    if (date) {
+      const parsedDate = new Date(date);
+      if (isNaN(parsedDate.getTime())) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)